### PR TITLE
v1.2 : major review 

### PR DESCRIPTION
**Pull request by paguardiolle**

### CODE REVIEW

File: src/advanced/eam/nullmx.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
 # Null MX record
 
-Basically the "null MX" protocol is a simple mechanism by which a domain can indicate that it does not accept email. It is described in [RFC 7505].
+The "null MX" protocol is used to state that mail services are disable in a given domain. The protocol is described in [RFC 7505].
```
Comment (ltabis) [inline src/advanced/eam/nullmx.md:3]: "used to state that mail services are `disabled` in a given domain."

File: src/advanced/eam/spf.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
 # Sender Policy Framework (SPF)
 
-This document specifies the vSMTP implementation of the Sender Policy Framework (SPF) protocol described in [RFC 7208](https://www.rfc-editor.org/rfc/rfc7208.html).
+This document describes the vSMTP implementation of the Sender Policy Framework (SPF) protocol described in [RFC 7208](https://www.rfc-editor.org/rfc/rfc7208.html).
 
-SPF is an authentication standard for linking a domain name and an email address. it allows email clients to verify that incoming email from a domain comes from a host authorized by the administrator of this domain.
+SPF is an authentication standard to link a domain name and an email address. it allows email clients to verify that incoming email from a domain comes from a host authorized by the administrator of this domain.
```
Comment (ltabis) [inline src/advanced/eam/spf.md:5]: "SPF is an authentication standard `used` to link a domain name and an email address"

File: src/advanced/toml.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 # vSMTP configuration files
 
-vSMTP and its sub-systems use [TOML] language for their configuration files. TOML files are frequently compared to INI for their similarities in syntax and use as configuration files.
+The configuration files of vSMTP and its sub-systems are defined using the [TOML] language. 
 
 [TOML]: https://toml.io/
 
-TOML uses tables (hash tables) as collections of key/value pairs. Key/value pairs within tables are not guaranteed to be in any specific order. Tables appear in square brackets on a line by themselves. Dots are used to signify nested tables. Nested array of tables are also allowed.
+TOML uses hash tables as collections of key/value pairs. Key/value pairs within tables are not supposed to be in any specific order. Tables appear in square brackets. Dots are used to signify nested tables. Nested array of tables are also allowed.
```
Comment (ltabis) [inline src/advanced/toml.md:7]: "Key/value pairs within tables `do not need` to be in any specific order"

File: src/advanced/toml.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
 ## The vsmtp.toml file
 
-This is the main configuration file. It should be located in `/etc/vsmtp`. However it can be modified in the systemd's service file `/etc/systemd/system/vsmtp.service` or using the `--config` option in interactive mode.
+The `vsmtp.toml` is the main configuration file. It should be located in `/etc/vsmtp` directory. An alternative filename or location can be specified in the systemd's service file `/etc/systemd/system/vsmtp.service` or, in interactive mode, using the `--config` option.
```
Comment (ltabis) [inline src/advanced/toml.md:11]: "It should be located in `the` /etc/vsmtp directory."

File: src/advanced/toml.md [hunk 5]
```diff
@@ -78,8 +78,8 @@ Please refer to the [reference guide] for a fully description of the key/value p
...
 ## The vsmtp.service file
 
-This file contains all the mandatory information to start the vSMTP service on Linux using [systemd] as the system and service manager.
+This file contains all the mandatory information to start the vSMTP service on Linux server using [systemd] as the system and service manager.
 
 [systemd]: https://freedesktop.org/wiki/Software/systemd/
 
-&#9762; | Do not modify this file unless you know what you are doing.
\ No newline at end of file
+&#9762; | Do not modify this file unless being highly aware of the impacts of your changes.
```
Comment (ltabis) [inline src/advanced/toml.md:85]: "unless `you are` aware of the impacts of your changes."

File: src/start/configuration/delegation.md [hunk 6]
```diff
@@ -96,10 +102,9 @@ import "service" as svc;
...
-Compromised emails are quarantined in the `virus_q` folder.
+> Since there is no heavy network traffic, John decided to do a post-queue filtering. Compromised emails are quarantined in the `virus_q` folder.
 
-## That's it
+Once the "check email for virus" rule is run, vsmtp will send the email to the clamsmtpd service and rule evaluation is on hold. Once all results are received on the delegation port (10025), evaluation
+resumes, and the body of this rule is evaluated.
 
-Any service that supports the SMTP protocol can be used to delegate the email
-processing / security with the `delegate` directive.
+___That's all folks !___
```
Comment (ltabis) [inline src/start/configuration/delegation.md:110]: Please remove that

File: src/start/configuration/hardening.md [hunk 7]
```diff
@@ -44,19 +46,17 @@ Now Doe's family server is protected against open-relaying attacks.
 
 ## Using the SPF protocol
 
-> You can find more information about SPF protocol in the [advanced section].
+> You can find more information about the SPF protocol in the [advanced section].
 
 [advanced section]: ../../advanced/eam/spf.md
 
-To allow other MTAs to verify that outgoing email from Doe's family domain comes from its server, we need to enable the SPF protocol. This is done by adding a new DNS text record that only allows only the MX record to send a mail for doe-family.com.
+The SPF protocol allows other MTAs to check that outgoing messages from Doe's family domain are valid. A new DNS record is added into the `doe-family.com` DNS zone. It announces that only the server declared in the MX record is allowed to send messages on behalf of Doe's family.
```
Comment (ltabis) [inline src/start/configuration/hardening.md:53]: "It `declares` that only the server declared in the MX record is allowed to send messages on behalf of Doe's family."

**Review by ltabis — CHANGES_REQUESTED** [review]

File: src/advanced/toml.md (hunk 5 above)
Comment (paguardiolle) [reply src/advanced/toml.md:85]: je suis en train de virer tous les "you"...donc forme passive.

**Review by ltabis — APPROVED** [review]

**Pull request merged**